pkg/spoke/apply/server_side_apply_test.go

```diff
@@ -0,0 +1,95 @@
+package apply
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	fakedynamic "k8s.io/client-go/dynamic/fake"
+	clienttesting "k8s.io/client-go/testing"
+	workapiv1 "open-cluster-management.io/api/work/v1"
+	"open-cluster-management.io/work/pkg/spoke/spoketesting"
+)
+
+func TestServerSideApply(t *testing.T) {
```

Comment:
Maybe these server-side apply unit tests are meaningless because the fake client doesn't support PatchType ApplyPatchType

Reply:
Not sure if we need to delete this test file?

Reply:
added an reactor to mock apply patch